bitnami/thanos/values.yaml

```diff
@@ -3217,6 +3217,12 @@ storegateway:
   ## Sharded parameters
   ## @param storegateway.sharded.enabled Enable sharding for Thanos Store Gateway
   ## @param storegateway.sharded.hashPartitioning.shards Setting hashPartitioning will create multiple store statefulsets based on the number of shards specified using the hashmod of the blocks
+  ## @param storegateway.sharded.hashPartitioning.customRelabelConfig Setting custom relabel config
+  ## e,g:
+  ## customRelabelConfig: 
+  ##   - action: keep
+  ##     source_labels: ["region"]
+  ##     regex: cn-zhangjiakou
```

Comment:
Should we name this parameter `extraRelabelingConfigs` instead? I think it'd better describe its purpose.

Reply:
I am glad to see your response, and I have already submitted new changes as per your suggestions.